src/WebJobs.Script/Workers/IWorkerChannel.cs

```diff
@@ -10,6 +11,8 @@ public interface IWorkerChannel
     {
         string Id { get; }
 
+        IWorkerProcess WorkerProcess { get; }
```

Comment:
Just want to confirm no one outside of this repo consumes this library. Adding members to interfaces is a breaking change.

Reply:
No one consumes the library.